File: app/src/main/java/com/anhhoang/popularmovies/model/GenreResponse.java
/*
 * Copyright (C) 2013 The Android Open Source Project
 */
/**
 * Created by AnhHo on 2/4/2017.
 */
package com.anhhoang.popularmovies.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class GenreResponse {
    @SerializedName("genres")
    private List<Genre> genres;

    public List<Genre> getGenres() {
        return genres;
    }
}

File: MovieApp/app/src/main/java/com/anhhoang/popularmovies/model/GenreResponse.kt
/*
 * Copyright (C) 2013 The Android Open Source Project
 */
/**
 * Created by AnhHo on 2/4/2017.
 */
package com.anhhoang.popularmovies.model

import com.google.gson.annotations.SerializedName

class GenreResponse(
        @SerializedName("genres")
        val genres: List<Genre>
)
